build/Targets/Settings.props

```diff
@@ -167,6 +174,7 @@
         <RemoveIntegerChecks>true</RemoveIntegerChecks>
         <RootNamespace></RootNamespace>
         <MSBuildTargetsLanguageName>VisualBasic</MSBuildTargetsLanguageName>
+        <NoWarn>$(NoWarn);40057</NoWarn>
```

Comment:
Here is the history of this change:

1. The old SDK disabled it everywhere.
1. The new SDK did not and I had to globally override it.
1. A random version of VS broke my override in 2 and a developer incorrectly checked in a whole bunch of suppressions
1. I fixed the global suppression and undid 3. 

So basically this is the old way of doing things 😦